samples/pom.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>aem-authoring-toolkit</artifactId>
+        <groupId>com.exadel.aem</groupId>
+        <version>1.2.1-SNAPSHOT</version>
+    </parent>
+    <properties>
+        <aat.version>1.2.1-SNAPSHOT</aat.version>
+    </properties>
+
+    <artifactId>aem-authoring-toolkit-samples</artifactId>
+    <name>AEM Authoring Toolkit - Samples</name>
+    <version>1.2.1-SNAPSHOT</version>
+
+    <packaging>pom</packaging>
+
+    <repositories>
+        <!-- For AEM Authoring Toolkit snapshot version -->
+        <repository>
+            <id>em-authoring-toolkit-snapshots</id>
+            <name>Toolkit Snapshot</name>
+            <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
+        </repository>
+    </repositories>
```

Comment:
Adobe's repo must reside here as well, because uber-jar is technically not guaranteed to be cached locally

Reply:
Added adobe repo